File: program/convert-fahrenheit-to-kelvin/convert_fahrenheit_to_kelvin.go
package main

import "fmt"

func main() {
	var fahrenheit float64
	fmt.Print("Enter temperature in Fahrenheit: ")
	fmt.Scanln(&fahrenheit)

	kelvin := (fahrenheit + 459.67) * 5 / 9
	fmt.Printf("%.3f Kelvin\n", kelvin)
}

File: program/convert-fahrenheit-to-kelvin/convert_fahrenheit_to_kelvin.hs
main :: IO ()
main = do 
    putStrLn "Enter temperature in Fahrenheit: " 
    input <- getLine
    let fahrenheit = read input :: Float

    let kelvin = (fahrenheit - 32) * 5/9 + 273.15
    putStrLn $ "Given Fahrenheit in Kelvin: " ++ show(kelvin) ++ " K"